auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/service/AltusIAMService.java

```diff
@@ -15,33 +17,67 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AltusIAMService.class);
 
+    private final boolean useSharedAltusCredential;
+
+    private final String accessKey;
+
+    private final char[] secretKey;
```

Comment:
Not a big deal, but I suggest renaming these by adding the prefix `sharedAltus` (or just `shared` to match the getters of `AltusDatabusConfiguration`).